.gitlab-ci.yml

```diff
@@ -3,10 +3,28 @@ stages:
   - unit-tests
   - dev
   - promote-to-stg
+  - common-stg
   - stg
```

Comment:
(Github won't let me add this comment where I want it.)

At some point (maybe right now :)) it sounds like you'll want to enable the (currently disabled) `gcp-stg` and `gcp-prd` Jobs.